### PR TITLE
Stronger typing

### PR DESCRIPTION
**Pull request by supersonicclay**

### CODE REVIEW

File: src/geojson-types.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// @flow
```
Comment (georgios-uber) [inline src/geojson-types.js]: Did you write this? I think there is a package you can import.

File: src/lib/layers/editable-geojson-layer.js [hunk 2]
```diff
@@ -11,9 +11,9 @@ import destination from '@turf/destination';
 import bearing from '@turf/bearing';
 import pointToLineDistance from '@turf/point-to-line-distance';
 import { point, featureCollection as fc } from '@turf/helpers';
-import type { GeoJsonFeature } from '../../types';
-import { EditableFeatureCollection } from '../editable-feature-collection';
-import EditableLayer from './editable-layer';
+import type { Feature } from '../../geojson-types.js';
```
Comment (georgios-uber) [inline src/lib/layers/editable-geojson-layer.js]: Remove .js ! (Linter did not complain?)

Comment (supersonicclay) [reply src/lib/layers/editable-geojson-layer.js]: In my opinion (and several others), having `.js` is more standards-compliant.

Examples:

* Chrome requires the `.js` extension when using `<script type="module"></script>` [reference](https://stackoverflow.com/a/44481876/444917)
* Ryan Dahl regretted allowing omitting the extension for node.js requires [video](https://youtu.be/M3BM9TB-8yA?t=13m57s)

So, unless there's a compelling reason to leave them off, I'd vote to keep extensions.

File: src/geojson-types.js (hunk 1 above)
Comment (supersonicclay) [reply src/geojson-types.js]: I looked at that, but nebula only supports a subset of GeoJSON (e.g. no `GeometryCollection`). Using our own types means we don't have to do `if (type !== 'GeometryCollection')` all over the place.

File: src/lib/layers/editable-geojson-layer.js (hunk 2 above)
Comment (georgios-uber) [reply src/lib/layers/editable-geojson-layer.js]: OK.

File: src/geojson-types.js (hunk 1 above)
Comment (georgios-uber) [reply src/geojson-types.js]: ok, great.

**Pull request merged**